neo/Network/P2P/Payloads/ConsensusPayload.cs

```diff
@@ -90,6 +93,7 @@ void IVerifiable.DeserializeUnsigned(BinaryReader reader)
             PrevHash = reader.ReadSerializable<UInt256>();
             BlockIndex = reader.ReadUInt32();
             ValidatorIndex = reader.ReadUInt16();
+            reader.ReadUInt32();
             Data = reader.ReadVarBytes();
             _deserializedMessage = ConsensusMessage.DeserializeFrom(Data);
```

Comment:
This will have an issue with older payloads. Consider #610

Reply:
For some reason the updated commit wasn't showing up in Github for me until later.